spec/v2/index.bs

```diff
@@ -493,14 +493,21 @@ Advisement:
         <td>String
         <td>M
         <td>
-            The IPCC version of the GWP characterization factors used in the calculation of the PCF (see [=Pathfinder Framework=] Section 3.2.2). The value MUST be one of the following:
+          **Deprecated**
+
+          The IPCC version of the GWP characterization factors used in the calculation of the largest relative share of the PCF (see [=Pathfinder Framework=] Section 3.2.2). The value MUST be one of the following:
 
           : `AR6`
           :: for the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (IPCC)
           : `AR5`
           :: for the Fifth Assessment Report of the IPCC.
 
-          Advisement: The set of characterization factor identifiers will likely change in future revisions. It is recommended to account for this when implementing the validation of this property.
+          Advisement: This property is deprecated and only kept to ensure backwards-compatibility. It will be removed on version 3 of these Technical Specifications. It does not replace the (also mandatory) property <{CarbonFootprint/characterizationFactorsSources}>.
+      <tr>
+        <td><dfn>characterizationFactorsSources</dfn>  : [=CharacterizationFactorsSourcesSet=]
+        <td>Array
+        <td>M
+        <td>The set of characterization factors used in the calculation of the PCF (see [=Pathfinder Framework=] Section 3.2.2). The value MUST be a [=CharacterizationFactorsSourcesSet=].
```

Comment:
As the Pathfinder Framework recommends the characterization factors used "shall be derived from the latest Intergovernmental Panel on Climate Change (IPCC) Assessment Report publication", we should make clear in the tech specs that the preference is to use only one, i.e. the latest. 

A potential amendment could be "The characterization factor(s) used in the calculation of the PCF (see [=Pathfinder Framework=] Section 3.2.2). Per the Framework, the latest available characterization factor version should be used. In the event this is not possible, include the set of all characterization factors used"